Modix.Services/Quote/MessageLinkBehavior.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Modix.Services.Quote
+{
+    public class MessageLinkBehavior : BehaviorBase
+    {
+        private static readonly Regex Pattern = new Regex(
+            @"https://(canary\.)?discordapp\.com/channels/(?<GuildId>\d+)/(?<ChannelId>\d+)/(?<MessageId>\d+)",
```

Comment:
It appears as though regular `http` also works, so maybe make the `s` optional

Reply:
Official links from the client will always be `https`, but yeah.